tools/gen_addon_readme.template

```diff
@@ -64,6 +64,7 @@ Authors
 Maintainers
 ~~~~~~~~~~~
 
+{% if org_name == 'OCA' -%}
```

Comment:
nice to have: if not OCA, use an alternative description passed as parameter or better define the footer (OCA or other) in external file so that generation is generic

Reply:
indeed, but not in the scope of this PR. Feel free to open an issue for such improvements.